test/riak_search_schema_tests.erl

```diff
@@ -0,0 +1,100 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(riak_search_schema_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-define(DEFAULT_DISABLED_JOB_CLASSES, [
```

Comment:
Looks like the `DEFAULT_DISABLED_JOB_CLASSES` define is not actually used anywhere. Should we delete this?